pkg/apis/apis/v1alpha1/types.go

```diff
@@ -378,6 +384,14 @@ type APIExportPolicy struct {
 // of the API Export
 type LocalAPIExportPolicy struct{}
 
+type PermissionClaim struct {
+	metav1.GroupResource `json:","`
+
+	// This is the identity for a given API Resource, It will be empty for core types.
```

Comment:
It's the identity of the APIExport that the APIResourceSchema belongs to

Reply:
updated.